Microsoft.Toolkit.Uwp.UI.Controls/Carousel/Carousel.cs

```diff
@@ -234,6 +234,11 @@ private void FocusContainerFromIndex(int index)
         }
 
         /// <summary>
+        /// Occurs when the selected item has changed
+        /// </summary>
+        public event EventHandler<object> SelectionChanged = null;
```

Comment:
Do you have examples of controls that offer selection and do not use this pattern?